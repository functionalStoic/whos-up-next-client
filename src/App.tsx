import React from 'react';
import './App.css';

function App() {
  return (
    <div className="App">
      <header className="App-header">
        <h1>Who's Up Next</h1>
        <p>Keep track of your important relationships</p>
      </header>
    </div>
  );
}

export default App;
