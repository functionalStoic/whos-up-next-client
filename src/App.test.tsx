import React from 'react';
import { cleanup, render } from 'react-testing-library';
import App from './App';

afterEach(cleanup);

test('renders without crashing', () => {
  const { container } = render(<App />);
  expect(container.firstChild).toMatchSnapshot();
});
