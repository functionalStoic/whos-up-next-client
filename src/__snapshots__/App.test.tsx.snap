// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders without crashing 1`] = `
<div
  class="App"
>
  <header
    class="App-header"
  >
    <h1>
      Who's Up Next
    </h1>
    <p>
      Keep track of your important relationships
    </p>
  </header>
</div>
`;
